### PR TITLE
Added: CLI arg --verifySsl

### PR DESCRIPTION
**Pull request by tfokken**
Allows the user to specify whether SSL certs are valid when making rest
calls to StreamSets

### CODE REVIEW

File: sdctool/api.py [hunk 1]
```diff
@@ -49,143 +50,148 @@ def start_pipeline(url, pipeline_id, auth, runtime_parameters={}):
...
 
-    poll_pipeline_status(STATUS_STOPPED, url, pipeline_id, auth)
+    poll_pipeline_status(STATUS_STOPPED, url, pipeline_id, auth, verify_ssl)
 
     logging.info('Pipeline stopped.')
     return stop_result.json()
 
 
-def validate_pipeline(url, pipeline_id, auth):
+def validate_pipeline(url, pipeline_id, auth, verify_ssl = True):
```
Comment (afoerster) [inline sdctool/api.py]: do you need the default arg here?

Comment (afoerster) [reply sdctool/api.py]: isn't `verify_ssl` always passed in (and it has a default)

**Comment by bpmcd** [comment]
Does it seem like 'verifySsl' will change on a per command basis?  Or will it be global for a particular installation?  If the answer is "global" then I think it would be better to put this flag in the config template/file.

**Comment by tfokken** [comment]
Agreed, adding it as a global in the config file makes sense.

**Comment by afoerster** [comment]
+1 to arg in the config file

**Pull request merged**